frontend/src/components/header.js

```diff
@@ -42,65 +50,63 @@ BootstrapNavLink.propTypes = {
  * @param {object[]} props.routes
  * @returns
  */
+
 export function Header(props) {
     const { routes = [], infoComponents = null } = props;
+    const [key, setKey] = React.useState(routes[0].route);
 
     if (routes.length === 0) {
         return <div>No Routes in Header</div>;
     }
-
     return (
-        <Navbar bg="light" variant="light">
-            <Navbar.Brand>
-                <NavDropdown
-                    title={routes.map((route) => (
-                        <Route path={route.route} key={route.route}>
-                            {route.name}
-                        </Route>
-                    ))}
-                >
-                    {routes
-                        .filter((route) => !route.hidden)
-                        .map((route) => (
-                            <NavDropdown.Item
-                                key={route.route}
-                                as="span"
-                                tabIndex="0"
+        <Tab.Container activeKey={key} onSelect={(k) => setKey(k)}>
+            <Row className="justify-content-between right-padding">
```

Comment:
I don't like these classnames. It's one thing if you are building a general framework for widget layout (which you shouldn't be, because that's the point of bootstrap...), but I think in this situation, they should be widget-specific. E.g., `nav-bar-item`

Reply:
The `justify-content-between` is a builtin bootstrap classname.
`right-padding` -- I agree it's annoying, I need to dig through the react-bootstrap classnames documentation to figure out how to give this element right padding.